src/filters/DropDown.jsx

```diff
@@ -0,0 +1,156 @@
+import React, {useState, useRef, useCallback, useMemo} from 'react';
+
+// export type DropDownItemType = {
```

Comment:
no types?

Reply:
same as the enum, saving it for a day we will write ts here :)